src/fun/counting/main.go.tmpl

```diff
@@ -0,0 +1,89 @@
+{{/*
+        Counting command. Count and have fun!
+        
```

Comment:
In [Contributing](https://github.com/yagpdb-cc/yagpdb-cc/blob/master/CONTRIBUTING.md), the link here is a https://yagpdb-cc.github.io link. As I do not have an entry on the CC website, I am unsure of what to put here; link to [Overview](https://github.com/H1nr1/yagpdb-cc/blob/master/website/docs/fun/counting/overview.md)?

Reply:
The URL is of the following structure:

```
domain/category/subfolder
```

You can construct the correct URL from the following information you already provided:

* The category is `fun`
* The subfolder is `counting`

This is given by the paths of the files you added.


Though I would recommend to rename the folder to counting-v2, so that it is distinct from the already existing counting custom command.